db/models/user.js

```diff
@@ -1,25 +1,30 @@
 'use strict'
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
-const bcrypt = require('bcryptjs')
-const Sequelize = require('sequelize')
-const db = require('APP/db')
+const bcrypt = require('bcryptjs');
+const Sequelize = require('sequelize');
+const db = require('APP/db');
 
 const User = db.define('users', {
   name: Sequelize.STRING,
   email: {
     type: Sequelize.STRING,
     validate: {
 			isEmail: true,
-			notEmpty: true,
-		}
+			notEmpty: true
+		},
+    unique: true
   },
 
   // We support oauth, so users may or may not have passwords.
   password_digest: Sequelize.STRING, // This column stores the hashed password in the DB, via the beforeCreate/beforeUpdate hooks
-	password: Sequelize.VIRTUAL // Note that this is a virtual, and not actually stored in DB
+	password: Sequelize.VIRTUAL, // Note that this is a virtual, and not actually stored in DB
+  isAdmin: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  }
 }, {
-	indexes: [{fields: ['email'], unique: true,}],
```

Comment:
Emily also mentioned that trailing commas are a Bones convention. I think they look silly too.